dashbuilder/dashbuilder-client/dashbuilder-widgets/src/main/java/org/dashbuilder/client/widgets/dataset/editor/prometheus/PrometheusDataSetEditor.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.client.widgets.dataset.editor.prometheus;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.Dependent;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+
+import org.dashbuilder.client.widgets.common.LoadingBox;
+import org.dashbuilder.client.widgets.dataset.editor.DataSetDefColumnsFilterEditor;
+import org.dashbuilder.client.widgets.dataset.editor.DataSetDefPreviewTable;
+import org.dashbuilder.client.widgets.dataset.editor.DataSetEditor;
+import org.dashbuilder.client.widgets.dataset.editor.attributes.DataSetDefBackendCacheAttributesEditor;
+import org.dashbuilder.client.widgets.dataset.editor.attributes.DataSetDefBasicAttributesEditor;
+import org.dashbuilder.client.widgets.dataset.editor.attributes.DataSetDefClientCacheAttributesEditor;
+import org.dashbuilder.client.widgets.dataset.editor.attributes.DataSetDefRefreshAttributesEditor;
+import org.dashbuilder.client.widgets.dataset.event.ErrorEvent;
+import org.dashbuilder.client.widgets.dataset.event.TabChangedEvent;
+import org.dashbuilder.common.client.editor.ValueBoxEditor;
+import org.dashbuilder.dataset.client.DataSetClientServices;
+import org.dashbuilder.dataset.def.PrometheusDataSetDef;
+
+/**
+ * <p>Prometheus Data Set editor presenter.</p>
+ * 
+ */
+@Dependent
+public class PrometheusDataSetEditor extends DataSetEditor<PrometheusDataSetDef> implements org.dashbuilder.dataset.client.editor.PrometheusDataSetDefEditor {
```

Comment:
Can you import `org.dashbuilder.dataset.client.editor.PrometheusDataSetDefEditor` instead of using its fully qualified name?